Makefile.in

```diff
@@ -64,6 +64,7 @@ GCC_MULTILIB_FLAGS := $(MULTILIB_FLAGS) --with-multilib-generator="$(MULTILIB_GE
 endif
 GLIBC_MULTILIB_NAMES := @glibc_multilib_names@
 GCC_CHECKING_FLAGS := @gcc_checking@
+GCC_TARGET_FLAGS := $(GCC_TARGET_FLAGS_EXTRA)
```

Comment:
Rename to `GCC_EXTRA_CONFIGURE_FLAGS`, `GCC_TARGET_FLAGS` is kind of confusing.
and you didn't add corresponding configure option flags, so you don't need the later part (`$(GCC_TARGET_FLAGS_EXTRA)`).